R/calibrate_sim.R

```diff
@@ -165,7 +165,7 @@ calibrate_sim <- function(var = 'temp',
   nml <- set_nml(nml, arg_list =total.list)
   write_nml(nml,nml_file)
   
-  run_glm(sim_folder = path, verbose = verbose)
+  run_glmcmd(glmcmd, path, verbose)
```

Comment:
does this rely on a certain version of GLM3r? if so, should add that as a version minimum in the `DESCRIPTION` file

Reply:
my understanding is that it should work with every version of GLM as at the moment it's only reading standard nml arguments (time, initial values) and reading temperature from ncdf output

Reply:
ok - I just don't see that function `run_glmcmd` anywhere. Am I missing it?

Reply:
it's in calib_helpers.R line 150. it allows the user e.g. to run a system call ('glm') or call a batch file, if another version of GLM is needed instead of GLM3r.

Reply:
gotcha. didn't see that.